Gems/Terrain/Code/Source/Components/TerrainPhysicsColliderComponent.cpp

```diff
@@ -298,6 +316,22 @@ namespace Terrain
         numRows = aznumeric_cast<int32_t>((bounds.GetMax().GetY() - bounds.GetMin().GetY()) / gridResolution.GetY());
     }
 
+    int32_t TerrainPhysicsColliderComponent::GetHeightfieldGridColumns() const
+    {
+        const AZ::Vector2 gridResolution = GetHeightfieldGridSpacing();
+        const AZ::Aabb bounds = GetHeightfieldAabb();
+
+        return aznumeric_cast<int32_t>((bounds.GetMax().GetX() - bounds.GetMin().GetX()) / gridResolution.GetX());
+    }
+
+    int32_t TerrainPhysicsColliderComponent::GetHeightfieldGridRows() const
+    {
+        const AZ::Vector2 gridResolution = GetHeightfieldGridSpacing();
+        const AZ::Aabb bounds = GetHeightfieldAabb();
+
+        return aznumeric_cast<int32_t>((bounds.GetMax().GetY() - bounds.GetMin().GetY()) / gridResolution.GetY());
+    }
+
```

Comment:
Suggestion:  Same as above, it might be worth implementing in terms of GetHeightfieldGridSize(), so that the calculation logic is at least only in one place.

Reply:
Done.